website/docs/docs/use-dbt-semantic-layer/exports.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "Exports"
+description: "Use Exports to materialize tables to the data platform on a schedule."
+sidebar_label: "Materialize with Exports"
+---
+
+Exports in the dbt Semantic Layer extends the [Saved Queries](/docs/build/saved-queries) functionality:
+
+- Exports enable you to materialize these queries within your data platform using dbt Cloud.
+- Exports provide an integration path for tools that don't nativley support the dbt Semantic Layer by exposing tables of metrics and dimensions.
+
+While Saved Queries are a way to save and reuse commonly used queries in MetricFlow, Exports takes this a step further by materialize these queries in your data platfrom. Exports are like any other table in your data platform, and allow you to query metric defintions in any SQL interface, or connect to downstream tools without needing a first class Sematnic Layer integration. 
+
+## Exports vs. Saved Queries
+
+| Feature    | Exports  | Saved Queries  |
+|-----------|-----------|----------------|
+| **Availability**    | Available to dbt Cloud users on [Team or Enterprise](https://www.getdbt.com/pricing/) plans, on dbt versions 1.7 or higher.| Available in both dbt Core and dbt Cloud.     |
+| **Purpose**         | To materialize Saved Queries in your data platform, and expose metrics and dimensions as a view or table. | To define and manage common Semantic Layer queries in YAML, which includes metrics and dimensions.   |
+| **Usage**           | Automatically runs Saved Queries and materializes them within your data platform. Query metric defintions using SQL or integrate with a BI tool that doesn't have a native integration. Exports count towards [queried metrics](/docs/cloud/billing#what-counts-as-a-queried-metric) usage. <br /><br />**Example**: Creating a weekly aggregated table for active user metrics, automatically updated and stored in the data platform.  | Used for organizing and reusing common MetricFlow queries within dbt projects.<br /><br />**Example**: Group related metrics together for better organizations, and include commonly uses dimensions and filters. | For materializing query results in the data platform. |
+| **Integration**     | Must have the dbt Semantic Layer configured in your dbt project.<br /><br />Tightly integrated with the [MetricFlow Server](/docs/use-dbt-semantic-layer/sl-architecture#components) and dbt Cloud's job scheduler. | Integrated into dbt DAG and managed alongside other dbt nodes. |
+| **Configuration**   | Configured within dbt Cloud environment and job scheduler settings. | Defined in YAML format within dbt project files.   |
```

Comment:
for SQ - am i right in thinking the only way to use them is via their editor or command line?